### DISCUSSION TITLE
gpui Macro for Templates

### DISCUSSION THREAD

**Discussion by matthiasg** [zed-industries/zed#8042] [Feature Requests]
I recently stumbled over GPUI and was wondering whether there is any discussion (could not find it in search) on creating a template syntax macro for it ?

gpui already looks very much like a simplified HTML / tailwind chain and using a macro would make it very comfortable for people coming to Rust from the web to work with.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
